test/test-types.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const Test = require('tape');
-var { buildASTSchema } = require('graphql');
+const { buildASTSchema } = require('graphql');
```

Comment:
Since we don't really need `graphql-tag`'s caching for excludes, should we just be building our own AST to handle the excludes and the issue with `graphql-tag` becomes moot?

Reply:
that's an interesting idea